app/src/main/java/is/xyz/mpv/MPVActivity.kt

```diff
@@ -345,6 +345,12 @@ class MPVActivity : Activity(), EventObserver, TouchGesturesObserver {
 
     override fun dispatchKeyEvent(ev: KeyEvent): Boolean {
         showControls()
+        if (ev.action == KeyEvent.ACTION_DOWN && interceptKeyDown(ev)) {
```

Comment:
This probably deserves another command similar to what you have in interceptKeyDown, e.g. "try a built-in event handler first, forward unhandled events to libmpv"